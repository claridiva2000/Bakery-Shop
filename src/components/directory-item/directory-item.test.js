import React from 'react';
import renderer from 'react-test-renderer';
import DirectoryItem from './directory-item.component';

it('should render DirectoryItem component', () => {
  const component = renderer.create(<DirectoryItem />).toJSON();
  expect(component).toMatchSnapshot();
});
