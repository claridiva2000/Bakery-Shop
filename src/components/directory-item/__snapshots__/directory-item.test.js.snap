// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DirectoryItem component 1`] = `
<div
  className="directory-item"
>
  <div
    className="image"
  >
    <span />
  </div>
  <h3 />
</div>
`;
